app/web/features/messages/groupchats/GroupChatView.tsx

```diff
@@ -84,29 +67,13 @@ export const useGroupChatViewStyles = makeStyles((theme) => ({
     },
   },
   pageWrapper: {
-    [theme.breakpoints.up("sm")]: {
-      height: `calc(100vh - ${theme.shape.navPaddingSmUp})`,
-    },
-    [theme.breakpoints.down("sm")]: {
-      height: `calc(100vh - ${theme.shape.navPaddingXs})`,
-    },
+    height: `calc(100vh - ${theme.shape.navPaddingSmUp})`,
     alignItems: "stretch",
     display: "flex",
     flexDirection: "column",
-    // 56px = address bar height on mobile - https://dev.to/peiche/100vh-behavior-on-chrome-2hm8
-    height: `calc(100vh - ${theme.shape.navPaddingXs} - 56px - 80px)`,
-  },
-  title: {
-    flexGrow: 1,
-    width: "100%",
-    display: "flex",
-    alignItems: "center",
-    marginInlineEnd: theme.spacing(2),
-    marginInlineStart: theme.spacing(2),
-    "& > *": { marginInlineEnd: theme.spacing(2) },
 
     [theme.breakpoints.down("sm")]: {
-      fontSize: "0.9rem",
+      height: `calc(100vh - ${theme.shape.navPaddingXs} - 80px)`, // 80 is space for mobile browser url bar
```

Comment:
This is what actually fixed it. If you have a non iPhone I especially want you to test this though. I'm not totally sure if the url bar sizing will be weird on some phones.